arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -52,11 +52,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
         
         self.modifyAppearance()
         
-        //enable/disable touches based on settings
+        // Enable/disable touches based on settings.
         self.setTouchPref()
         
         SVProgressHUD.setDefaultMaskType(.gradient)
         
+        // Set license key with custom build rule.
+        AppDelegate.licenseApplication()
```

Comment:
Not really sure what will happen after the metered API comes in. If it means the developer mode doesn't work anymore, we should have this line; otherwise we should comment this line out and add docstring such as Uncomment the line below to..."